ESP_Code/Settings.h

```diff
@@ -14,7 +14,7 @@ extern const char RIG_IDENTIFIER[] = "None";
 
 // Uncomment if you want to use LAN8720
 // Select correct Board in ArduinoIDE!!! Really!
- #define USE_LAN
+// #define USE_LAN
```

Comment:
Good that you made it off by default, but I'd move it from general settings to somewhere lower since it's something that a minority will use